src/jsMain/kotlin/SignupComponent.kt

```diff
@@ -1,20 +1,204 @@
+import io.ktor.util.valuesOf
 import react.*
 import react.dom.*
 import kotlinext.js.*
+import kotlinx.browser.document
 import kotlinx.html.js.*
 import kotlinx.coroutines.*
 import kotlinx.css.*
+import smallComponents.InputComponent
 import styled.*
+import org.w3c.dom.HTMLSelectElement
+import org.w3c.dom.events.Event
 
 //Do we need this line below?
 //private val scope = MainScope()
 
 val SignupPage = functionalComponent<RProps> { _ ->
     val (username, setUsername) = useState("")
     val (password, setPassword) = useState("")
-    val (reEnterPassword, setReEnterPassword) = useState("")
+    val (email, setEmail) = useState("")
+    val (Zipcode, setZipcode) = useState("")
```

Comment:
Can you make this zipcode and setZipcode in order to match the standard set elsewhere

Reply:
Got it